src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -5156,7 +5156,9 @@ public partial class IndentedTextWriter : System.IO.TextWriter
         [System.Diagnostics.CodeAnalysis.AllowNullAttribute]
         public override string NewLine { get { throw null; } set { } }
         public override void Close() { }
+        public override ValueTask DisposeAsync() => throw null;
```

Comment:
Oops, types need to have fully qualified name in ref
```suggestion
        public override System.Threading.Tasks.ValueTask DisposeAsync() => throw null;
```

Reply:
Sorry about that. These "ref" projects are uncharted territory for me. I'll fix them and add the new unit tests.

Reply:
No problem at all, thank you for working on it